src/applications/personalization/profile/tests/e2e/notification-settings/happy-loading-path.cypress.spec.js

```diff
@@ -1,7 +1,6 @@
 import { PROFILE_PATHS } from '@@profile/constants';
 
-import mockPatient from '@@profile/tests/fixtures/users/user-36.json';
-import mockNonPatient from '@@profile/tests/fixtures/users/user-non-patient.json';
+import { makeUserObject } from '~/applications/personalization/common/helpers';
```

Comment:
switching the mock user object that's used in these tests required updating a few existing assertions that check for the existence of phone number and email address in the DOM.